.github/workflows/release-arm.yaml

```diff
@@ -230,21 +230,36 @@ jobs:
           files: |
             build/*scan-reports.tar.gz
       - name: Prepare sarif files  🔧
+        if: startsWith(github.ref, 'refs/tags/')
         run: |
-          mkdir trivy-sarif grype-sarif
-          sudo mv build/*trivy.sarif trivy-sarif/
-          sudo mv build/*grype.sarif grype-sarif/
+          mkdir trivy-results grype-results
+          trivy=$(ls build/*trivy.sarif 2>/dev/null | head -n 1)
+          grype=$(ls build/*grype.sarif 2>/dev/null | head -n 1)
+          sudo mv $trivy trivy-results/result.sarif
+          sudo mv $grype grype-results/result.sarif
+      - uses: itxaka/sarif-filter@v1
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          input: trivy-results/result.sarif
+          output: trivy-results/result.sarif
+          severity: high
+      - uses: itxaka/sarif-filter@v1
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          input: grype-results/result.sarif
+          output: grype-results/result.sarif
```

Comment:
umm, I havent tested this but input and output being the same may result in broken files if its being read while being written.

Reply:
I tried it locally and it works. It first loads the json into memory and then writes back to the output file without a care in the world. Should be ok.